personal_tokens.yaml

```diff
@@ -33,7 +33,31 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/PersonalTokenResponseWithResults'          
+                $ref: '#/components/schemas/PersonalTokenResponseWithResults'
+
+  /iam/personal_tokens/{personalTokenId}:
+    get:
+      tags:
+        - Personal Token
+      summary: Get a personal token info
+      description: Get a personal token info
+      operationId: GetPersonalToken
+      parameters: 
+      - in: path
+        name: personalTokenId
+        required: true
+        schema:
+          format: uuid
+          type: string
+      
+      responses:
+        '200':
```

Comment:
400, 404 and 500 can also be responded, so you need to list them here.

Reply:
Fixed in: https://github.com/aziontech/azionapi-openapi/pull/70/commits/df5585c50cfb63c2fdeffbf8c7e00c40ac6834f1